cmd/app/manifest.go

```diff
@@ -48,7 +48,9 @@ func resolveVariables(manifestContent []byte, vars map[string]string) ([]byte, e
 		err  error
 		b    bytes.Buffer
 	)
-	if tmpl, err = template.New("").Parse(string(manifestContent)); err != nil {
+	tplFuncMap := make(template.FuncMap)
+	tplFuncMap["Split"] = strings.Split
```

Comment:
We should do this like Hugo and use package as namespace (strings.Split). But let's not complicate this right now. All good.